databases/week1/hw-week1.sql

```diff
@@ -0,0 +1,53 @@
+USE hyf;
+-- 1Find out how many tasks are in the task table
+SELECT COUNT(id)
+ FROM task;
+ 
+-- 2  find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(id) 
+FROM task
+WHERE due_date IS null;
+
+-- 3  Find all the tasks that are marked as done
+SELECT title 
+FROM task 
+WHERE status_id = 3;
+
+-- 4  Find all the tasks that are not marked as done
+SELECT title 
+FROM task 
+WHERE status_id != 3;
+
+-- 5 Get all the tasks, sorted with the most recently created first
+SELECT title 
+FROM task 
+order by created;
+
+-- 6 Get the single most recently created task
```

Comment:
I am not sure I understand this query :) Remember in class you talked about the sql `LIMIT` clause? 

    SELECT title FROM task LIMIT 1;

will return only the first row found. What do you need to do to the list of rows before using `LIMIT`? Maybe use the previous exercise as inspiration.

Try to be consistent with the formatting

    SELECT title, MAX(created) the_finla_task
    FROM task; 

where each clause is on its own line.